src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -321,26 +321,36 @@ private bool CanInstallPackage()
         {
             if (DisplayRoots != null)
             {
-                foreach (PackageItem packageItem in DisplayRoots)
+                CheckIfFilesAreSelected(DisplayRoots);
+            }
+
+            return !_isInstalling && !NoFilesSelected;
+        }
+
+        private void CheckIfFilesAreSelected(IReadOnlyList<PackageItem> children)
+        {
+            foreach (PackageItem child in children)
+            {
+                if (child.Children.Any())
```

Comment:
>Any() [](start = 35, length = 5)

No need for linq, right?

Reply:
Checking with count instead.

---
In reply to: [194554437](https://github.com/aspnet/LibraryManager/pull/126#discussion_r194554437) [](ancestors = 194554437)